src/api/app/views/webui2/webui/project/_project_inherited_packages.html.haml

```diff
@@ -0,0 +1,14 @@
+.card-body
+  - if ipackages.present?
+    %table.table.table-striped.table-bordered#inherited-packages-table
+      %thead
+        %tr
+          %th Name
+          %th From
+      %tbody
+        - ipackages.each do |ipackage|
+          %tr
+            %td
+              = link_to(ipackage[0], package_show_path(package: ipackage[0], project: ipackage[1]))
```

Comment:
[nitpick] I would use `.first` for readability, then you can forget if the array starts in 0 or 1.